release-notes/swan-lake-beta2-release-note.md

```diff
@@ -36,6 +36,16 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
 
 #### Improvements
 
+The static type of string iteration has been changed from `string` to `string:Char`.
+
+public function main() {
+    string str = "foo";
+
+    foreach string:Char s in str {
+        
+    }
+}
```

Comment:
````suggestion
```ballerina
public function main() {
    string str = "foo";

    foreach string:Char s in str {
        io:println(s);        
    }

    record {| string:Char value; |}? next = str.iterator().next();

    if !(next is ()) {
        io:println(next.value);
    }
}
```
````